test/unit/org/apache/cassandra/index/sai/cql/DBPE14965Test.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Nit: I think we need the DS header

Reply:
what do you mean? I can see the same header in many other files

Reply:
This is the header that comes from ASF and that we maintain in the files with modify. But I think new files created by us should have [this other header](https://github.com/datastax/cassandra/blob/main/src/java/org/apache/cassandra/db/SortedLocalRanges.java#L1-L15).

Reply:
Updated